### PR TITLE
Strings moved from source code to resources

### LINKED ISSUE #94: Move texts in sources to resorces
We have some string directly in sources (in Utils even in slovak language). We need to move it into resources.

### PR DESCRIPTION
**Pull request by satano**
Closes #94

### CODE REVIEW

File: Kros.Utils/src/Kros.Utils/Properties/Resources.resx [hunk 1]
```diff
@@ -117,79 +117,94 @@
...
   </data>
-  <data name="DatabaseSchemaLoader_UnsupportedConnectionType" xml:space="preserve">
-    <value>Nepodporovaný typ spojenia na databázu.</value>
+  <data name="IBulkInsertFactoryNotRegisteredForConnection" xml:space="preserve">
+    <value>{0} for connection type "{1}" is not registered.</value>
   </data>
-  <data name="IndexSchema_CannotChangeIndexNameWhenBelongsToTable" xml:space="preserve">
-    <value>Nie je možné zmeniť názov indexu, ak je v zozname indexov tabuľky. Tento index patrí tabuľke "{0}".</value>
+  <data name="IdGeneratorFactoryNotRegisteredForClient" xml:space="preserve">
+    <value>IdGeneratorFactory for ADO client {0} is not registered.</value>
```
Comment (Burgyn) [inline Kros.Utils/src/Kros.Utils/Properties/Resources.resx]: somewhere we have something in the sense of `class name is not registered ...` and elsewhere `{0} is not registered ...`. Can we unify it?

File: Kros.Utils/src/Kros.Utils/Properties/Resources.resx [hunk 2]
```diff
@@ -117,79 +117,94 @@
...
-    <value>Hodnota nemôže byť null.</value>
+    <value>The value cannot be null.</value>
   </data>
   <data name="Check_StringNotNullOrEmpty" xml:space="preserve">
-    <value>Reťazec nemôže byť null, alebo prázdny.</value>
+    <value>The value cannot be null, neither empty string.</value>
   </data>
   <data name="Check_StringNotNullOrWhiteSpace" xml:space="preserve">
-    <value>Reťazec nemôže byť null, prázdny, ani zložený iba z bielych znakov.</value>
+    <value>The value cannot be null, nither empty string, nor string containing only whitespace characters.</value>
```
Comment (Burgyn) [inline Kros.Utils/src/Kros.Utils/Properties/Resources.resx]: - nither -> neither
- nor -> or?

File: Kros.Utils/src/Kros.Utils/Properties/Resources.resx [hunk 3]
```diff
@@ -117,79 +117,94 @@
...
   </data>
-  <data name="Schema_ColumnBelongsToAnotherTable" xml:space="preserve">
-    <value>Stĺpec "{0}" nie je možné pridať tabuľke "{1}", pretože už patrí tabuľke "{2}".</value>
+  <data name="MissingPropertyInType" xml:space="preserve">
+    <value>Type "{0}" does not have property "{1}" which is in column names list.</value>
   </data>
-  <data name="Schema_ForeignKeyBelongsToAnotherTable" xml:space="preserve">
-    <value>Cudzí kľúč "{0}" nie je možné pridať tabuľke "{1}", pretože už patrí tabuľke "{2}".</value>
+  <data name="PathContainsInvalidCharacters" xml:space="preserve">
+    <value>One of the path parts contains invalid characters. Errorneous part is at index {0}: "{1}"</value>
```
Comment (Burgyn) [inline Kros.Utils/src/Kros.Utils/Properties/Resources.resx]: - Errorneous  -> Erroneous?

File: Kros.Utils/src/Kros.Utils.MsAccess/Data/BulkActions/MsAccess/MsAccessBulkInsert.cs [hunk 4]
```diff
@@ -368,8 +368,7 @@ private string CreateInsertSql(string tableName, string sourceFilePath)
 
         private string SqlCodePage
         {
-            get
-            {
+            get {
```
Comment (LukasSefcik) [inline Kros.Utils/src/Kros.Utils.MsAccess/Data/BulkActions/MsAccess/MsAccessBulkInsert.cs]: Many opening braces is on the same line as get keyword. It is on purpose?

Comment (satano) [reply Kros.Utils/src/Kros.Utils.MsAccess/Data/BulkActions/MsAccess/MsAccessBulkInsert.cs]: No. Something changed my settings in Visual Studio.

File: Kros.Utils/src/Kros.Utils/Properties/Resources.resx (hunk 2 above)
Comment (satano) [reply Kros.Utils/src/Kros.Utils/Properties/Resources.resx]: Will correct "nither". "nor" is good. ;)

**Pull request merged**